go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/roadrunner-server/sendremotefile
```

Comment:
It should be `v4`:
```suggestion
module github.com/roadrunner-server/v4/sendremotefile
```
All plugins should be in sync with the major version.

Reply:
I guess it should be `github.com/roadrunner-server/sendremotefile/v4`, at least that's the pattern I find in other plugins/middlewares

Reply:
Yeah, you're right 👍